nipype/utils/filemanip.py

```diff
@@ -430,3 +431,13 @@ def write_rst_dict(info, prefix=''):
     for key, value in sorted(info.items()):
         out.append(prefix + '* ' + key + ' : ' + str(value))
     return '\n'.join(out)+'\n\n'
+
+
+def mkdir_p(path):
+    try:
+        os.makedirs(path)
```

Comment:
even with the try is on the creation aspect only? because the only error is EEXISTS according to the initial issue.